src/app/series/directives/series-feeds.component.html

```diff
@@ -25,10 +25,43 @@ <h2>No feeds found! Please contact support</h2>
           <button *ngIf="!expanded[i]" type="button" class="expand" aria-label="Expand" (click)="expand(i)"></button>
         </header>
         <section *ngIf="expanded[i]">
-          <prx-fancy-field *ngIf="!f.isDefault" required textinput [model]="f" name="title" id="title{{ i }}" label="Feed Label">
-            <div class="fancy-hint">A label for this feed, such as "Limited Items" or "Apple Subscription"</div>
+          <prx-fancy-field *ngIf="!f.isDefault" label="Overrides" required>
+            <div class="fancy-hint">
+              Override series metadata for this feed. You must provide an alternate title, but the other fields are optional.
+            </div>
+
+            <prx-fancy-field required textinput [model]="f" name="title" id="title{{ i }}" label="Title" small>
+            </prx-fancy-field>
+
+            <prx-fancy-field *ngIf="f.subtitle" textinput [model]="f" name="subtitle" id="subtitle{{ i }}" label="Teaser" small>
+            </prx-fancy-field>
+
+            <prx-fancy-field *ngIf="f.description" label="Description" small>
+              <publish-wysiwyg [model]="f" name="description" [content]="f.description" [changed]="f.changed('description')"></publish-wysiwyg>
+            </prx-fancy-field>
+
+            <prx-fancy-field *ngIf="f.summary" label="Summary">
+              <publish-wysiwyg [model]="f" name="summary" [content]="f.summary" inputFormat="HTML" outputFormat="HTML" [changed]="f.changed('summary')"></publish-wysiwyg>
+            </prx-fancy-field>
+
+            <p *ngIf="!f.subtitle">
+              <input type="checkbox" (click)="f.set('subtitle', ' ')" id="alternateSubtitle{{ i }}">
+              <label for="alternateSubtitle{{ i }}">Enter an alternate teaser</label>
+            </p>
+
+            <p *ngIf="!f.description">
+              <input type="checkbox" (click)="f.set('description', ' ')" id="alternateDescription{{ i }}">
+              <label for="alternateDescription{{ i }}">Enter an alternate description</label>
+            </p>
+
+            <p *ngIf="!f.summary">
+              <input type="checkbox" (click)="f.set('summary', ' ')" id="alternateSummary{{ i }}">
+              <label for="alternateSummary{{ i }}">Enter an alternate summary</label>
+            </p>
           </prx-fancy-field>
 
+          <hr>
```

Comment:
This `<hr>` appears to be showing up on the default feed about the FIle Name. Looks awkward there, but necessary for the other fields. 

Maybe it needs to be inside the `<prx-fancy-field>`??